integration-tests/tests/src/tests/results.ts

```diff
@@ -171,48 +171,6 @@ describe("Results", () => {
       }).throws("Realm object is from another Realm");
     });
 
-    it("implements addListener", function (this: RealmContext) {
```

Comment:
I figured this is worth removing now... Much cleaner way to deal with this with `done` too.